src/diagnostics/logPointsClient.ts

```diff
@@ -185,30 +223,47 @@ export class CommandRunResult<ResponseType extends { error?: {}, data?: {} }> {
     }
 }
 
-// tslint:disable-next-line:max-classes-per-file
+export class CloseSessionRequest {
+    constructor(public sessionId: string) {
+    }
+}
+
 export class AttachProcessRequest {
     constructor(public sessionId: string, public processId: string) {
     }
 }
 
-// tslint:disable-next-line:max-classes-per-file
 export class DebugSessionMetadata {
     constructor(public sessionId: string, public debugId: string) {
     }
 }
 
-// tslint:disable:max-classes-per-file
 export class LoadSourceRequest {
     constructor(public sessionId: string, public debugId: string, public sourceId: string) {
     }
 }
 
+export class SetLogpointRequest {
```

Comment:
Dont' need to address this now, but I'm not sure why these are classes - interfaces would be more flexible here, as `{ sessionId, debugId, sourceId}` is duck-type compatible with your class LoadSourceRequest.

Reply:
It's done as well.